src/main/java/seedu/address/logic/commands/PictureCommand.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_FILE_LOCATION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Name;
+
+/**
+ * Adds a picture to a contact in the address book.
+ */
+public class PictureCommand extends Command {
+
+    public static final String COMMAND_WORD = "pic";
+    public static final String COMMAND_ALIAS = "p";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a picture to a contact in the address book "
+        + "by the name used in the displayed person list.\n"
```

Comment:
For now, this should still be "by the index number used in the displayed person list.\n", editing people by name hasn't and won't be implemented by v1.1, we should maintain consistency with the rest of the codebase currently.